legacy/core/src/main/java/com/fsck/k9/preferences/SettingsExporter.kt

```diff
@@ -482,11 +483,10 @@ class SettingsExporter(
     fun generateDatedExportFileName(): String {
         val now = Calendar.getInstance()
         val dateFormat = SimpleDateFormat("yyyy-MM-dd", Locale.US)
-        return String.format("%s_%s.%s", EXPORT_FILENAME_PREFIX, dateFormat.format(now.time), EXPORT_FILENAME_SUFFIX)
+        return String.format("%s_%s.%s", filePrefixProvider.filePrefix, dateFormat.format(now.time), EXPORT_FILENAME_SUFFIX)
```

Comment:
Now, instead of `thunderbird_settings_export_2024-10-22.k9s`, we'll end up with `thunderbird_2024-10-22.k9s`.

```suggestion
        return String.format("%s_%s_%s.%s", EXPORT_FILENAME_MIDDLE_PART, filePrefixProvider.filePrefix, dateFormat.format(now.time), EXPORT_FILENAME_SUFFIX)
```

With
```kotlin
companion object {
    private const val EXPORT_FILENAME_MIDDLE_PART = "settings_export"
    // …
}
```

Reply:
@cketti there is an issue with argument order. Fixed it now.